### PR TITLE
Adds more information regarding camera settings

### PR DESCRIPTION
**Pull request by Crafty15**
- Added another table with our currently used camera settings.
- Added links to the camera information page and the Spinnaker SDK.
- Added a collapsable list section with all of the possible camera settings. This includes a link to the arv-tool command line utility used to print out the settings.
- Updated the information regarding how the settings are applied.
- Added a line on how to use the command line utility to get camera settings.

### CODE REVIEW

**Comment by LachlanCourt** [comment]
Don't forget to run `./b yarn format` , should fix up the build error 😊

File: src/book/02-system/02-subsystems/01-input.mdx [hunk 1]
```diff
@@ -24,8 +28,963 @@ The cameras have the following parameters that are used by the object detection
...
+
+  ```
+
+
+</details>
+
+
+
+
 The [projection tool](https://github.com/NUbots/NUbots/blob/main/shared/utility/vision/projection.hpp), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
```
Comment (ysims) [inline src/book/02-system/02-subsystems/01-input.mdx:989]: This is repeated above.

**Review by ysims — COMMENTED** [review]
Don't forget to add yourself to the authors list.

**Review by LachlanCourt — APPROVED** [review]
LGTM

File: src/book/02-system/02-subsystems/01-input.mdx [hunk 2]
```diff
@@ -24,8 +29,959 @@ The cameras have the following parameters that are used by the object detection
...
-The [projection tool](https://github.com/NUbots/NUbots/blob/main/shared/utility/vision/projection.hpp), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
+The [projection tool](https://github.com/NUbots/NUbots/tree/main/shared/utility/vision/projection.h), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
+
+<details id="settings_list">
+
+  <summary>Click here to see a full list of camera settings</summary>
+
+**This file is a print out directly from the cameras using [arv-tool-0.8](https://github.com/AravisProject/aravis)**
+
+**This file is a print out directly from the cameras using the [arv-tool-0.8](https://github.com/AravisProject/aravis) command line utility.**
```
Comment (ysims) [inline src/book/02-system/02-subsystems/01-input.mdx:59]: repeated line?

File: src/book/02-system/02-subsystems/01-input.mdx [hunk 3]
```diff
@@ -24,8 +29,959 @@ The cameras have the following parameters that are used by the object detection
...
 
 The parameters are used in the **Camera module** to find and set up the cameras. The Camera module emits [Image messages](https://github.com/NUbots/NUbots/tree/main/shared/message/input/Image.proto).
 
-The [projection tool](https://github.com/NUbots/NUbots/blob/main/shared/utility/vision/projection.hpp), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
+The [projection tool](https://github.com/NUbots/NUbots/tree/main/shared/utility/vision/projection.h), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
```
Comment (ysims) [inline src/book/02-system/02-subsystems/01-input.mdx:51]: This is here twice? And the link in this version is wrong, it should be hpp still.

File: src/book/02-system/02-subsystems/01-input.mdx [hunk 4]
```diff
@@ -24,8 +29,959 @@ The cameras have the following parameters that are used by the object detection
...
+    Enumeration : 'TransferQueueMode' (Not available)
+        EnumEntry   : 'FirstInFirstOut'
+    Command     : 'TransferStart' (Not available)
+    Command     : 'TransferStop' (Not available)
+
+```
+
+</details>
+
+The [projection tool](https://github.com/NUbots/NUbots/tree/main/shared/utility/vision/projection.h), based on [panotools' fisheye projection calculations](https://wiki.panotools.org/Fisheye_Projection), maps a portion of the surface of a sphere to a flat image. The type of projection is specified by the above parameter `lens.projection`.
```
Comment (ysims) [inline src/book/02-system/02-subsystems/01-input.mdx:987]: Here's the second one - and it has a bad link.

**Review by ysims — CHANGES_REQUESTED** [review]

File: src/book/02-system/02-subsystems/01-input.mdx (hunk 3 above)
Comment (Bidski) [reply src/book/02-system/02-subsystems/01-input.mdx:51]: `https://github.com/NUbots/NUbots/tree/main/shared/utility/vision/projection.h` ????? i thought we renamed all of the `.h` files to `.hpp` in the repo ages ago?

Comment (Crafty15) [reply src/book/02-system/02-subsystems/01-input.mdx:51]: Good spot. Not sure how that happened

File: src/book/02-system/02-subsystems/01-input.mdx (hunk 2 above)
Comment (Crafty15) [reply src/book/02-system/02-subsystems/01-input.mdx]: :/

**Comment by ysims** [comment]
Might want to add in the parameters from https://github.com/NUbots/NUbots/pull/987

**Review by LachlanCourt — APPROVED** [review]

**Review by ysims — APPROVED** [review]
I'll add an issue for my last comment

**Pull request merged**